portfolio.html

```diff
@@ -0,0 +1,231 @@
+    <title>Beryl's Portfolio</title>
+    <link rel="stylesheet" href="style.css">
+  </head>
+  <body>
+    <header class="nav_section header">
+      <div class="my_logo">
+        <h2 class="logo"><a class="logo" href="#">GAKII </a></h2>
+      </div>
+      <nav>
+        <div class="menu">
+          <img src="./images/Union.png" class="log" />
+        </div>
+        <ul class="links">
+          <li><a href="#">Portfolio</a></li>
+          <li><a href="#">About</a></li>
+          <li><a href="#">Contact</a></li>
+          <li>
+            <a class="fa-regular fa-envelope"></a>
+          </li>
+        </ul>
+      </nav>
+    </header>
+    <section class="headline1">
+      <article>
+        <h1>I’m Gakii<br />Glad to see,You!</h1>
+        <p>
+          I’m a software developer! I can help you build a product , feature or
+          website Look through some of my work
+          and experience! If you like what you see and have a project you need
+          coded, don’t hestiate to contact me.
+        </p>
+        <a href="#">LET’S CONNECT</a>
+      </article>
+        <div class="social-media">
+          <ul>
+            <li><a href="#"><img src="images/github.svg" alt=""></a></li>
+            <li><a href="#"><img src="images/Linkedin%20icon.svg" alt=""></a></li>
+            <li><a href="#"><img src="images/medium.svg" alt=""></a></li>
+            <li><a href="#"><img src="images/github.svg" alt=""></a></li>
+            <li><a href="#"><img src="images/angellist.svg" alt=""></a></li>
+          </ul>
+        </div>
+    </section>
+    <section class="grid_container">
+      <div class="block1">
+        <div class="block1a">
+          <img src="images/Snapshoot Portfolio1.png" />
+        </div>
+          <div class="item1b">
+            <h2 class="tonic">Tonic</h2>
+            <div class="frame2 tonic">
+              <span>CANOPY</span>
+              <ul class="role">
+                <li>Back End Dev</li>
+                <li>2015</li>
+              </ul>
+            </div>
+            <p class="tonic">
+              A daily selection of privately personalized reads; no accounts or sign-ups required.
+            </p>
+            <div class="tags tonic">
+              <ul>
+                <li>html</li>
+                <li>css</li>
+                <li>javaScript</li>
+              </ul>
+              <button type="button"><a href="">See Project</a></button>
+            </div>
+        </div>
+      </div>
+      <div class="item2">
+        <div class="item2a">
+          <img src="images/Snapshoot%20Portfolio2.png">
+        </div>
+        <div class="item1b">
+          <h2>Multi-Post Stories</h2>
+          <div class="frame2">
+           <span>FACEBOOK</span>
+            <ul class="role">
+              <li>Full Stack Dev</li>
+              <li>2015</li>
+            </ul>
+          </div>
```

Comment:
- [ ] Kindly, make sure all project item match the figma design when resized.
**Your design**
![r1](https://user-images.githubusercontent.com/74411135/229210481-e647152a-34a3-49a1-902f-56e95c014ec3.png)
**The figma design**
![image](https://user-images.githubusercontent.com/74411135/229210678-238026b4-f6aa-422e-b363-cacb9eb1423f.png)

This will make your design prettier which will improve the user experience 📱 💻 🖥️ 😃 😺